packages/app/src/app/pages/Sandbox/Editor/Workspace/items/ProjectInfo/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { inject, observer } from 'mobx-react';
 
-import Project from '../../Project';
+// eslint-disable-next-line import/extensions
+import Project from '../../Project/index.tsx';
```

Comment:
@SaraVieira No need for the file extension here

```suggestion
import Project from '../../Project/index';
```

Reply:
Fixed